extensions/projection/README.md

```diff
@@ -0,0 +1,124 @@
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| proj:epsg        | integer\|null  | **Required** [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:crs         | string \|null  | Recommended. PROJ string representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:extent` fields represent |
+| proj:geometry    | Polygon Object | Recommended. Defines the footprint of this item. |
+| proj:extent      | [number]       | Recommended. Bounding box of the item or asset in the native CRS |
+| proj:centroid    | [number]       | Recommended. Coordinates representing the centroid of the item or asset in the native CRS |
+
+**proj:epsg** - A Coordinate Reference System (CRS) is the native reference system (sometimes called a
+'projection') used by the data, and can usually be referenced using an [EPSG code](http://epsg.io).
+If the data does not have a CRS, such as in the case of non-rectified imagery with Ground Control
+Points, proj:epsg should be set to null. It should also be set to null if a CRS exists, but for which
+there is no valid EPSG code.
+
+**proj:crs** - A Coordinate Reference System (CRS) is the native reference system (sometimes called a
+'projection') used by the data. This value is a PROJ string.
+If the data does not have a CRS, such as in the case of non-rectified imagery with Ground Control
+Points, proj:crs should be set to null. It should also be set to null if a CRS exists, but for which
+a PROJ string does not exist.
+
+**proj:geometry** - A Polygon object representing the footprint of this item, formatted according the Polygon object format specified in [RFC 7946, sections 3.1.6](https://tools.ietf.org/html/rfc7946), except not necessarily in EPSG:4326 as required by RFC7946.  Specified based on the `proj:crs` field (not necessarily EPSG:4326). Ideally, this will be represented by a Polygon with five coordinates, as the item in the native CRS should be a square aligned to the CRS grid.  It is recommended that either or both of `proj:geometry` and `proj:extent` be defined.
+
+**proj:extent** - Bounding box of the asset represented by this item in the native CRS. Specified as four coordinates based on the CRS defined in the `proj:epsg` and `proj:crs` fields.  First two numbers are coordinates of the lower left corner, followed by coordinates of upper right corner, e.g., \[west, south, east, north]. It is recommended that either or both of `proj:geometry` and `proj:extent` be defined.
```

Comment:
I'd consider further clarification/disambiguation on what it means to provide both `geometry` *and* `extent`. Does one take precedence? Is one expected to be more precise than another?

Because this spec is for imagery in a rectilinear grid and an extent + CRS precisely defines the coverage, and the approximate 4326 geometry is already covered in the parent spec, I'd suggest removing the `proj:geometry` field, and require the expression to be in extent form.

Reply:
You could also ask this in the item spec, which also has bbox and geometry. 

By the way, why is proj:extent not called proj:bbox to align with the GeoJSON fields in the item spec?

Reply:
There are also edge-cases where an image can be in a projected coordinate system but the extent and geometry don't match due to rotation in the geotransform, in which case both `proj:geometry` and `proj:extent` would be required to describe the spatial properties of the image.

Reply:
But in this extension, `proj:geometry` is defined in the native crs (`proj:crs`), so technically they are redundant. This is distinct from the GeoJSON case where the `bbox` is in EPSG:4326 and and a separate (yet still approximate)  `geometry` is needed due the aforementioned geotransform rotation (which is *not* an edge case in our use cases.)

So, in the Item spec you need at least a `geometry` to get the approximate footprint, and a `bbox` gives you a convenient and fast approximation. In the `proj` extension everything is in the native CRS, so (I'm assuming?) the footprint can be fully expressed as an extent/bbox (unless there's some spherical geometry condition I'm not considering).

Reply:
The difference in `geometry` and `bbox` you are referring to (such as seen in Landsat8 images) is not due to geotransform rotation but rather because the Landsat image is resampled onto a square array for ease-of-use.  If you pull the geotransform of any standard Landsat8 it will be 0, indicating no rotation -- even though visually there is rotation.

I believe you are making an assumption that images in projected coordinate systems always line up with the crs axis but this is only true when the image is not tilted.  When no tilt is present, the `bbox` and `geometry` will perfectly coincide (with a jaccard index of 1).  If there is tilt in the geotransform the array will not align with the crs axis, causing disparate `bbox` and `geometry` (with a jaccard index of < 1).  This is similar to the Landsat images mentioned above but its happening for a different reason.  

Because of this I think that the potential for `proj:geometry` and `proj:extent` to differ within a single item is a valid edge case across both geogcs and projcs.

Reply:
Thanks for explaining!